fetchai/ledger/api/tx.py

```diff
@@ -1,6 +1,33 @@
+import base64
+import binascii
+
 from .common import ApiEndpoint
 
 
+class TxStatus:
+    def __init__(self,
+                 digest: bytes,
+                 status: str,
+                 exit_code: int,
+                 charge: int,
+                 charge_rate: int,
+                 fee: int):
+        self._digest = digest
+        self.status = status
+        self.exit_code = exit_code
+        self.charge = charge
+        self.charge_rate = charge_rate
+        self.fee = fee
+
+    @property
+    def finished(self):
+        return self.status not in ('Unknown', 'Pending')
+
+    @property
+    def digest(self):
+        return binascii.hexlify(self._digest).decode()
```

Comment:
Since this is probably going to be accessed a few times, is it worth caching this do you think?

Reply:
True; done.